ui/app/scripts/app.js

```diff
@@ -23,6 +23,7 @@ define([
   'angular',
   './analytics/app',
   './job/app',
```

Comment:
Unless there is a specific reason, we can complete remove the mappings of `job` and associated resources (scripts, routes etc.,)?